### ISSUE TITLE
Build failure - Core

### ISSUE THREAD

**Issue by inallib** [abixen/abixen-platform#277]
Core module build failed. All the other modules are building properly and running. I have done a fetch upstrem and the a merge till checkin "Small fixes and improvements related to the comment domain".
I tried mvn clean and Install also but no luck. I got bellow stacktrace.

[ERROR] Failed to execute goal com.spotify:docker-maven-plugin:0.4.13:build (default) on project abixen-platform-core: Exception caught: java.util.concurrent.ExecutionException: com.spotify.docker.client.shaded.javax.ws.rs.ProcessingException: org.apache.http.conn.HttpHostConnectException: Connect to localhost:2375 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused: connect -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

**Comment by mariuszkumor** [comment]
Looks like you have missed -DskipDocker.

**Comment by sauravms** [comment]
Nope its there already and its working for other modules.

**Comment by inallib** [comment]
Nope -DskipDocker is there already.

**Comment by inallib** [comment]
here are the requested logs and screenshots
![modules - runner config](https://cloud.githubusercontent.com/assets/24612079/21483247/1c8d912a-cba5-11e6-9954-1c4fa1085b28.png)
![platform - install](https://cloud.githubusercontent.com/assets/24612079/21483248/1c903c36-cba5-11e6-8f96-1aab5b5c03be.png)

![core - runner config](https://cloud.githubusercontent.com/assets/24612079/21483249/1c9503ce-cba5-11e6-9af1-42fa984a5801.png)
[Modules - build result.txt](https://github.com/abixen/abixen-platform/files/673043/Modules.-.build.result.txt)

**Comment by alvin-reyes** [comment]
I would recommend doing a clean as well before the install.

`mvn clean install -DskipDocker`

**Comment by nagukothapalli** [comment]
@inallib 
did you execute this mvn clean install -DskipDocker at root level ?

**Comment by inallib** [comment]
Yep I did it all... even I have done mvn install -e that indicates Core MS was ignored because of errors.

**Comment by alvin-reyes** [comment]
I ran the entire build after getting the latest copy and it worked as expected.

Here are the steps that I did:
`mvn clean install -DskipDocker`

Run in this order:

- eureka
- configuration
- hystrix 
- gateway
- core
- module 
- web-client

When I ran into some issues, this is usually means that I didn't wait for other projects to finish it's initialization. This is specifically happen to me between core, module and web-client. I advise you to just run each one of them and wait for one to finish before running the next.
